### DISCUSSION TITLE
Switch a Plug and Play Project from Mainnet to Testnet

### DISCUSSION THREAD

**Discussion by kallinlobus** [Web3Auth/Web3Auth#1140] [Help]
Is it possible to switch a plug and play project on the Web3Auth dashboard from Mainnet to Testnet? Now that the network is being enforced in the adapter settings, I realized I need my project to be on testnet in the Plug and Play settings. But I don't want to make a new plug and play project because I don't want to lose the wallets that I was using on testnet in my current project.

**Comment by shahbaz17** [comment]
Hey @kallinlobus 

It is recommended to use the Mainnet Plug n Play project's client id for production applications.

Let me check internally and see if this can be done or not.

Also, here it's unclear whether you want to move from mainnet to testnet or vice-versa. 
> I don't want to lose the wallets I used on testnet in my current project.
